threading_utils.py

```diff
@@ -0,0 +1,3 @@
+def propagate_exceptions(futures):
+    # Calling result() on a future whose execution raised an exception will propagate the exception to the caller
+    [future.result() for future in futures]
```

Comment:
nit: can we add a new line at the end of the file?

Reply:
fixed https://github.com/databrickslabs/migrate/pull/142/commits/dcad022083961b4826befbf23376145b1641896f